_gsocproposals/2020/proposal_IntAlertSystem.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Intelligent Alert Management System For HEP experiments
+layout: gsoc_proposal
+project: Monitoring
```

Comment:
The project name cannot be just "Monitoring", but maybe CMS-monitoring, or much better just CMS if this is a CMS-driven development, since I've learnt today from @dpiparo that there will be several proposals this year. You need to add a file _gsocprojects/2020/CMS(or whatever).md with a general description for CMS (or monitoring subsystem), please follow examples under _gsocprojects/2019/. Please include a project logo named CMS(or whatever project name)_logo.png under /images, and reference it in CMS.md